src/components/Technologies/PacksFilters.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import styles from "./PackFilters.module.scss";
+import filterStyles from "./CategorySelector/CategorySelector.module.scss";
+import CustomLabel from "./CategorySelector/CustomLabel";
+import AdditionalFilters from "./CategorySelector/AdditionalFilters";
+import { packTypeNames, cloudProviderTypes } from "./PackConstants";
+import { Select } from "antd";
+interface PackFiltersProps {
+  categories: string[];
+  registries: any[];
+  setSelectedSearchFilters: (...args: any[]) => void;
+}
+export default function PacksFilters({ categories, registries, setSelectedSearchFilters }: PackFiltersProps) {
+  function selectAdditionalFilters(additionalFilters: string[]) {
+    setSelectedSearchFilters({ additionalFilters: additionalFilters });
+  }
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.filterItems}>
+        <CustomLabel label="Type" />
+        <div className={filterStyles.wrapper}>
+          <Select
+            className={filterStyles.selectbox}
+            mode="multiple"
+            allowClear
+            placeholder="Search"
+            onChange={(item) => setSelectedSearchFilters({ category: item as string[] })}
+          >
+            {categories.map((category) => {
+              return (
+                <Select.Option key={category}>
+                  {packTypeNames[category as keyof typeof packTypeNames]}
+                </Select.Option>
+              );
+            })}
+          </Select>
```

Comment:
create a component for this and reuse it bellow as well. remove unnecessary code duplication